modules/ipp/scanner.go

```diff
@@ -409,8 +417,12 @@ func (scanner *Scanner) Grab(scan *scan, target *zgrab2.ScanTarget, version *ver
 		}
 		if strings.HasPrefix(strings.ToUpper(p), "CUPS/") {
 			scan.results.CUPSVersion = p
-			/*err :=*/ scanner.augmentWithCUPSData(scan, target, version)
-			// TODO: Log error
+			err := scanner.augmentWithCUPSData(scan, target, version)
+			if err != nil {
+				scan.log.WithFields(log.Fields{
+					"error": err,
+				}).Warn("Failed to augment with CUPS-get-printers request.")
```

Comment:
Since these scans get run over the entire public IPv4 space, we typically would only want info+-level logs to appear when there is a problem with the scan itself, rather than a problem with an individual host.

The exception would be if debugging / verbose logging is enabled. So, I see two easy options -- move all of the `*.Warn()` etc to `*.Debug()`, or, set the log level of your logger in such a way that it only logs if `Flags.Verbose` is set (I think `scan.log.SetLevel(logrus.ErrorLevel)` should work, but I'm not certain).